src/Swashbuckle.AspNetCore.SwaggerGen/SwaggerGenerator/SwaggerGenerator.cs

```diff
@@ -328,6 +330,12 @@ private IList<OpenApiTag> GenerateOperationTags(ApiDescription apiDescription)
 
         private IList<OpenApiParameter> GenerateParameters(ApiDescription apiDescription, SchemaRepository schemaRespository)
         {
+            if (apiDescription.ParameterDescriptions.Any(apiParam => IsFromFormAttributeUsedWithIFormFile(apiParam)))
```

Comment:
This can be simplified to not need the lambda:

```suggestion
            if (apiDescription.ParameterDescriptions.Any(IsFromFormAttributeUsedWithIFormFile))
```

Reply:
Thanks. Done!